packages/fxa-auth-server/scripts/write-emails-to-disk.js

```diff
@@ -111,6 +111,14 @@ function sendMail(mailer, messageToSend) {
     invoiceDate: new Date(),
     nextInvoiceDate: new Date(Date.now() + 1000 * 3600 * 24 * 30),
     serviceLastActiveDate: new Date(Date.now() + 1000 * 3600 * 24 * 60),
+    productIconNew: 'http://placekitten.com/512/512?image=1',
```

Comment:
Another nice to have for another PR, it would be nice to run this script on auth-server builds and upload rendered email as artifacts.

Reply:
Ah yeah, that would be a good idea. I'll add an issue to the epic and maybe tackle it in this next sprint

Reply:
Made followup: https://github.com/mozilla/fxa/issues/4950

Reply:
https://jira.mozilla.com/browse/FXA-1641